.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,3 +13,5 @@ Resolves #
 - [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
 - [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
 - [ ] `make vendor` does not cause changes.
+- [ ] Godoc comments have been added to (1) new code additions, and (2) existing code which has been
+      sufficiently modified. See: https://blog.golang.org/godoc-documenting-go-code
```

Comment:
Instead of linking to the godocs directly how about linking to either CONTRIBUTING.md or a (new) markdown file which is just for "code design/style guide rules"?

The "code design/style guide rules" would then contain links to the given format/style specifications Rook team and community has agreed on (e.g. for Golang default `go fmt` usage and adding comments to functions and complex logics).

Though I think for now adding a section to CONTRIBUTING.md should be enough and handling the move of that section to a general "style" guide in another issue would be okay.